fe1-web/src/features/evoting/screens/CreateElection.tsx

```diff
@@ -141,22 +153,46 @@ const CreateElection = () => {
   );
   const [electionName, setElectionName] = useState<string>('');
 
-  const [questions, setQuestions] = useState<NewQuestion[]>([EMPTY_QUESTION]);
+  const [questions, setQuestions] = useState<NewQuestion[]>(defaultQuestions);
   const [version, setVersion] = useState<ElectionVersion>(ElectionVersion.OPEN_BALLOT);
 
   // UI state
   const [modalEndIsVisible, setModalEndIsVisible] = useState<boolean>(false);
   const [modalStartIsVisible, setModalStartIsVisible] = useState<boolean>(false);
 
+  // Automatically compute the trimmed questions
+  const trimmedQuestions = useMemo(() => {
+    return questions
+      .map((item) => {
+        return {
+          ...item,
+          question: item.question.trim(),
+          ballot_options: item.ballot_options
+            .map((val) => val.trim())
+            .filter((value) => value !== ''),
+        };
+      })
+      .filter((question) => question.question !== '');
+  }, [questions]);
+
   // Confirm button only clickable when the Name, Question and 2 Ballot options have values
   const confirmButtonEnabled: boolean =
     isConnected === true &&
-    electionName !== '' &&
-    !questions.some(isQuestionInvalid) &&
-    haveQuestionsSameTitle(questions);
+    electionName.trim() !== '' &&
+    areQuestionsValid(trimmedQuestions) && // Checks there are enough valid questions to open an election
+    !questions.some(isNonEmptyQuestionInvalid) && // Checks that there are no questions that will be silently removed
+    !trimmedQuestions.some(areBallotOptionsInvalid) &&
```

Comment:
Picky detail but I think it could read even a bit nicer :)
```suggestion
    !trimmedQuestions.some(hasInvalidBallotOptions) &&
```